OpenSim/Simulation/Manager/Manager.h

```diff
@@ -56,15 +56,15 @@ class ControllerSet;
  * using setIntegrator().
  * 
  * In order to prevent an inconsistency between the Integrator and TimeStepper,
- * we only create a TimeStepper once, specifically at the first call to
- * integrate(SimTK::State&, double). To ensure this, the Manager will throw 
- * an exception if setModel() or setIntegrator() is called after 
- * integrate(SimTK::State&, double) has been called at least once.
+ * we only create a TimeStepper once, specifically when we call
+ * initialize(SimTK::State&). To ensure this, the Manager will throw
+ * an exception if initialize(SimTK::State&) is called more than once. Note 
+ * that any subsequent changes to the SimTK::State passed into 
+ * initialize(SimTK::State&) will not affect the simulation.
  *
- * Since the call to integrate(SimTK::State&, double) takes the state as an 
- * argument, it is up to the caller to ensure that the state is a legal state 
- * if the same Manager is used to integrate again. Integrating a different 
- * state for some new arbitrary system has undefined behavior.
+ * Note that this interface means that you cannot "reinitialize" a Manager.
+ * If you make changes to the SimTK::State, a new Manager must be created
+ * before integrating again.
```

Comment:
These comments are really helpful. I think we should mention the specific situation that Lukasz ran into; see my comments on `Manager::integrate()`.